### ISSUE TITLE
crates.io

### ISSUE THREAD

**Issue by oli-obk** [solson/miri#412]
Ping @bgeron

Are you willing to give up the `miri` crate on crates.io? We're thinking about publishing the miri binary and `cargo-miri` via crates.io to allow ppl to easily check deterministic code for the cases of UB that we can detect

**Comment by bgeron** [comment]
Absolutely! I presume you want the cargo-miri package as well. What's your username on crates.io? Or on the internals forum for direct messaging?

**Comment by bgeron** [comment]
Apparently `cargo-miri` has already been downloaded 99 times so you're getting a winner! ;)

**Comment by oli-obk** [comment]
:D cool, thanks

crates.io usernames and github usernames are the same. So "oli-obk"

**Comment by bgeron** [comment]
Two invites sent! Please remove me when you accept ownership :)

**Comment by RalfJung** [comment]
Thanks @bgeron :)

Can the same package have two names...?

**Comment by bgeron** [comment]
I don't think so. I registered two distinct packages. (But one package can contain multiple crates I believe, or at least multiple binaries.)
